Anki/public/assets/js/auth.js

```diff
@@ -0,0 +1,65 @@
+var firebaseConfig = {
```

Comment:
You'll have to put all JS into new PR as well.

Reply:
Removed all js code